tsl/test/sql/cagg_api.sql

```diff
@@ -177,10 +178,16 @@ SELECT hypertable_id,
   FROM _timescaledb_catalog.continuous_aggs_invalidation_threshold
   ORDER BY 1;
 
+-- Show the table definitions since there should be only one trigger
+-- or we will get duplicate entries.
+\d hyper_ts
+\d hyper_tstz
```

Comment:
Do we really need it?

Reply:
No. It is an attempt to capture a duplicate trigger, which I suspected was the cause of the duplicate entries. Since the [failing run did not show a diff here](https://github.com/timescale/timescaledb/actions/runs/14926005571/job/41934949204?pr=8076), we can remove it.

Reply:
Removed.